mod/node-api/handlers/beacon/types/request.go

```diff
@@ -60,7 +60,7 @@ type GetValidatorBalancesRequest struct {
 
 type PostValidatorBalancesRequest struct {
 	types.StateIDRequest
-	IDs []string `validate:"dive,validator_id"`
+	IDs []string `json:"-" validate:"dive,validator_id"`
```

Comment:
why is the json tag just a "-"?

Reply:
The request body in this case is an array. If we don't have a JSON tag altogether, it will be by default pick the field name, so ignoring it. I have added a comment to explain the reasoning as well for future reference.